ui/src/pages/namespace/Mirror/Detail/index.tsx

```diff
@@ -98,17 +104,13 @@ const Activities = () => {
                     <TableRow className="hover:bg-inherit dark:hover:bg-inherit">
                       <TableCell colSpan={4}>
                         <NoResult icon={GitCompare}>
-                          {t("pages.mirror.activities.list.noResults")}
+                          {t("pages.mirror.syncs.list.noResults")}
                         </NoResult>
                       </TableCell>
                     </TableRow>
                   )}
-                  {currentItems.map((activity) => (
-                    <Row
-                      namespace={data.namespace}
-                      key={activity.id}
-                      item={activity}
-                    />
+                  {currentItems.reverse().map((sync) => (
```

Comment:
I don't exactly know what problems you had with changing the order on react Query level, but I tried this and I did not see anything unusual:

**src/api/syncs/query/get.ts (line 43)**
`select: (data) => filter({ data: data.data.reverse() }),`

Note: I also remove the `reverse()` from the JSX

Reply:
Fixed in caf3c86540487a4472cfe175f1726724954c1f6e

As discussed, I needed to copy the array to avoid mutating the array in cache. I also moved this to the useListSyncs hook, because the order isn't relevant in other use cases.